modules/metering-uninstall.adoc

```diff
@@ -3,25 +3,34 @@
 // * metering/metering-uninstall.adoc
 
 [id="metering-uninstall_{context}"]
-= Uninstalling metering from {product-title}
+= Uninstalling metering from your namespace
 
-You can remove metering from your cluster.
+Uninstall metering from your namespace by removing the MeteringConfig resource and deleting the `openshift-metering` namespace.
 
 .Prerequisites
 
-* Metering must be installed.
+* The Metering Operator is removed from your cluster.
 
 .Procedure
 
-To remove metering:
-
-. In the {product-title} web console, click *Operators* -> *Installed Operators*.
-
-. Find the Metering Operator and click the {kebab} menu. Select *Uninstall Operator*.
-
-. In the dialog box, click *Remove* to uninstall metering.
-
-[NOTE]
-====
-Metering does not manage or delete any S3 bucket data. If you used Amazon S3 for storage, any S3 buckets used to store metering data must be manually cleaned up.
-====
+.  Remove all resources created by the Metering Operator:
++
+[source,terminal]
+----
+$ oc --namespace openshift-metering delete meteringconfig <1>
```

Comment:
Just want to confirm that this CLI command is correct. Normally it would look something like:

```
$ oc --namespace <namespace> delete <resource_type> <resource_name>
```

Or you might be able to delete all meteringconfig instances without specifying a name after, but then the note in the callout wouldn't be correct that it's the name of the single resource.

Reply:
> Just want to confirm that this CLI command is correct. Normally it would look something like:
> 
> ```
> $ oc --namespace <namespace> delete <resource_type> <resource_name>
> ```
> 
> Or you might be able to delete all meteringconfig instances without specifying a name after, but then the note in the callout wouldn't be correct that it's the name of the single resource.

@timflannagan1 What do you think about using the example in Andrea's comment (if correct) then having an example with the `openshift-metering` namespace and `meteringconfig` custom resource as example values?

Reply:
@lbarbeevargas I don't have an opinion on what's best here, but we explicitly call out somewhere in the docs we only support a single MeteringConfig custom resource in the namespace you installed metering in. 

If you want to do something like `oc --namespace <namespace> delete <resource_type> --all` that's sufficient too, but the expectation is that there's going to only be a single instance. Else, I think it's fine to use that suggestion instead of what we currently have documented in this PR.

Reply:
@lbarbeevargas My apologies, I realize now that I wasn't clear when I made my above suggestion.

I didn't actually mean to suggest to use what I put in my example code block. I was more trying to convey that I thought the command was missing something. That it should have both a resource type (meteringconfig?) and a resource name (mymeteringconfig, etc.) passed to the `oc delete` command.

So to summarize, if the following CLI command is correct, then please use that (and not my text above):

```
$ oc --namespace openshift-metering delete meteringconfig
```

But what I'm still unsure about is whether `meteringconfig` is the type, or the name. I would just ask that you confirm that the callout text that followed was accurate in saying that `meteringconfig` was the name, or if is actually the type. @timflannagan1 not sure if you can confirm this or not.

Reply:
Yeah, MeteringConfig is the resource type here. You still need to specify an instance of that type though as `oc --namespace openshift-metering delete meteringconfig` is invalid. 

You could do `oc --namespace openshift-metering delete meteringconfig <1>` where `<1>` maps to your meteringconfig name, e.g. `oc --namespace openshift-metering delete meteringconfig metering`.

Reply:
Or, alternatively, you could just do `oc --namespace openshift-metering delete meteringconfig --all` and that will achieve the same result.